packages/manager/src/managers/telemetry/types.ts

```diff
@@ -114,6 +117,11 @@ type OpenPageSnippetSegmentEvent = SegmentEvent<
 	{ framework: string }
 >;
 
+type CopyPageSnippetSegmentEvent = SegmentEvent<
+	typeof SegmentEventType.copy_page_snippet,
+	{ framework: string; repository: string }
```

Comment:
```suggestion
	{ framework: string; }
```

No need then finally